test/OpenTelemetry.Extensions.PersistentStorage.Tests/FileBlobProviderTests.cs

```diff
@@ -16,143 +16,146 @@
 
 using System;
 using System.IO;
-using System.Linq;
 using System.Text;
 using System.Threading;
 using Xunit;
 
 namespace OpenTelemetry.Extensions.PersistentStorage.Tests
 {
-    public class FileStorageTests
+    public class FileBlobProviderTests
     {
         [Fact]
-        public void FileStorage_E2E_Test()
+        public void FileBlobProvider_E2E_Test()
         {
             var testDirectory = new DirectoryInfo(Path.Combine(Path.GetTempPath(), Path.GetRandomFileName()));
-            using var storage = new FileStorage(testDirectory.FullName);
+            using var blobProvider = new FileBlobProvider(testDirectory.FullName);
 
             var data = Encoding.UTF8.GetBytes("Hello, World!");
 
             // Create blob.
-            IPersistentBlob blob1 = storage.CreateBlob(data);
+            Assert.True(blobProvider.TryCreateBlob(data, out var blob));
 
             // Get blob.
-            IPersistentBlob blob2 = storage.GetBlob();
+            Assert.True(blobProvider.TryGetBlob(out var singleBlob));
 
-            Assert.Single(storage.GetBlobs());
+            Assert.Single(blobProvider.GetBlobs());
 
             // Verify file name from both create blob and get blob are same.
-            Assert.Equal(((FileBlob)blob1).FullPath, ((FileBlob)blob2).FullPath);
+            Assert.Equal(((FileBlob)blob).FullPath, ((FileBlob)singleBlob).FullPath);
```

Comment:
nit: imo "blob" and "singleBlob" isn't descriptive of what's being tested here.

I think the original "blob1" and "blob2" are fine here.
Could also use "createBlob" and "getBlob".